src/tqec/detectors/transformer.py

```diff
@@ -46,45 +53,66 @@ def _fill_in_detectors_global_record_indices_impl(
                     index_of_last_explored_moment - current_moment_index
                 ) * operation_repetitions
                 operations.append(operation.replace(circuit=modified_circuit.freeze()))
-            elif isinstance(operation.gate, RelativeMeasurementGate):
-                if len(operation.qubits) > 1:
-                    raise MeasurementAppliedOnMultipleQubitsException(operation.qubits)
-                new_operation = deepcopy(operation)
-                relative_measurement_gate: RelativeMeasurementGate = new_operation.gate  # type: ignore
-                relative_measurement_gate.compute_global_measurements_lookback_offsets(
-                    global_measurement_map, current_moment_index
+                continue
+
+            # All the stim-related annotations should have been tagged with STIM_TAG
+            if STIM_TAG not in operation.tags:
+                operations.append(deepcopy(operation))
+                continue
+            untagged = operation.untagged
+            if isinstance(untagged, ShiftCoords):
+                operations.append(stimcirq.ShiftCoordsAnnotation(untagged.shift))
```

Comment:
```suggestion
                operations.append(stimcirq.ShiftCoordsAnnotation(untagged.shifts))
```
see previous comment.